server/src/containers/docker.ts

```diff
@@ -62,7 +62,7 @@ export async function attachContainer(
     Tty: true,
     Env: [
       `TOKEN=${token}`,
-      `API_ENDPOINT=${config.backendUrl}`,
+      `API_ENDPOINT=http://backend:3000`,
```

Comment:
change environment variables in your personal `.env` instead of hard coding. You may need to import `dotenv` for load `.env` files